### DISCUSSION TITLE
Trouble adding timestamps

### DISCUSSION THREAD

**Discussion by SpeckOfDust2545** [deepgram/community#722] [General help] [answered]
I've been using the API Playground site (https://playground.deepgram.com/) to transcribe YouTube videos. This works great, as does the diarization but I can't find an option to enable Timestamps. Anyone know how to do this? Thanks.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
This was answered in Discord: https://discord.com/channels/1108042150941294664/1232398067890917427
